pkg/config/env.go

```diff
@@ -209,4 +209,21 @@ var Config = struct {
 	// UI path  => localhost:18000/foo"
 	// API path => localhost:18000/foo/api/v1"
 	WebPrefix string `env:"FLAGR_WEB_PREFIX" envDefault:""`
+
+	// NotifyTimeout - for each notifier that uses http.Client to make requests
+	NotifyTimeout time.Duration `env:"FLAGR_NOTIFY_TIMEOUT" envDefault:"30"`
```

Comment:
it's duration, so the default value should be something like `30s` - assuming you are referring to 30s of duration.

Reply:
Totally right, I'll change that